tests/scripts/analyze_outcomes.py

```diff
@@ -207,6 +207,64 @@ def do_analyze_driver_vs_reference(outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_curves_except_p192': {
+        'test_function': do_analyze_driver_vs_reference,
+        'args': {
+            'component_ref': 'test_psa_crypto_config_reference_curves',
+            'component_driver': 'test_psa_crypto_config_accel_all_curves_except_p192',
+            'ignored_suites': [
+                # Following ones are skipped because these are the software
+                # implementations that are intentionally excluded from the 
+                # driver build, so disparities are expected here.
+                'ecdh',
+                'ecdsa',
+                'ecjpake',
+                'ecp',
+                # Following ones, instead, should be removed by #6622 follow up
```

Comment:
Ok, so it looks like there's quite a few. This makes me glad we isolated "curves" as an intermediate goal. Otherwise we would have discovered this only when attempting to remove ECP, which I'm sure will have its own set of challenges.